mmtrack/models/trackers/oc_sort.py

```diff
@@ -0,0 +1,478 @@
+# Copyright (c) OpenMMLab. All rights reserved.
+import lap
+import numpy as np
+import torch
+from addict import Dict
+from mmcv.runner import force_fp32
+from mmdet.core import bbox_overlaps
+
+from mmtrack.core.bbox import bbox_cxcyah_to_xyxy, bbox_xyxy_to_cxcyah
+from mmtrack.models import TRACKERS
+from .base_tracker import BaseTracker
+
+
+@TRACKERS.register_module()
+class OCSORT_Tracker(BaseTracker):
+    """Tracker for ByteTrack.
+
+    Args:
+        obj_score_thrs (float): Detection score threshold for matching objects.
+            Defaults to 0.3.
+        init_track_thr (float): Detection score threshold for initializing a
+            new tracklet. Defaults to 0.7.
+        weight_iou_with_det_scores (bool): Whether using detection scores to
+            weight IOU which is used for matching. Defaults to True.
+        match_iou_thr (float): IOU distance threshold for matching between two
+            frames. Defaults to 0.3.
+        num_tentatives (int, optional): Number of continuous frames to confirm
+            a track. Defaults to 3.
+        vel_consist_weight (float): Weight of the velocity consistency term in
+            association (OCM term in the paper).
+        vel_delta_t (int): The difference of time step for calculating of the
+            velocity direction of tracklets.
+        init_cfg (dict or list[dict], optional): Initialization config dict.
+            Defaults to None.
+
+    refer to mmtrack/models/mot/motion/kalman_filter.py for details.
+    """
+
+    def __init__(self,
+                 obj_score_thr=0.3,
+                 init_track_thr=0.7,
+                 weight_iou_with_det_scores=True,
+                 match_iou_thr=0.3,
+                 num_tentatives=3,
+                 vel_consist_weight=0.2,
+                 vel_delta_t=3,
+                 init_cfg=None,
+                 **kwargs):
+        super().__init__(init_cfg=init_cfg, **kwargs)
+        self.obj_score_thr = obj_score_thr
+        self.init_track_thr = init_track_thr
+
+        self.weight_iou_with_det_scores = weight_iou_with_det_scores
+        self.match_iou_thr = match_iou_thr
+        self.vel_consist_weight = vel_consist_weight
+        self.vel_delta_t = vel_delta_t
+
+        self.num_tentatives = num_tentatives
+
+    @property
```

Comment:
Consider inherit from sort_tracker or bytetrack_tracker?

Reply:
I have made it inherit from sort_tracker.